src/generators/operationGenerator.ts

```diff
@@ -277,6 +286,19 @@ function buildRequestBody({
 
   // First get the mapper, this will be the mapper for the parameter before flattening
   const mapper = requestBody[0].mapper;
+  let required: boolean = requestBody[0].required ?? false;
+
+  if (!required) {
+    for (const rb of requestBody) {
```

Comment:
Can you please add a comment about what this block is doing?

Reply:
Done